JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -0,0 +1,55 @@
+using Amazon;
+using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace JustSaying.Sample.Restaurant.KitchenConsole
+{
+    internal class Program
+    {
+        private static async Task Main()
```

Comment:
Nit: `private` seems odd to me for the main method of an app.

Reply:
done